src/main/java/com/projecty/projectyweb/message/MessageService.java

```diff
@@ -14,32 +16,36 @@
 import java.sql.Timestamp;
 import java.util.List;
 import java.util.Optional;
+import java.util.stream.Collectors;
 
 
 @Service
 public class MessageService {
-    private static UserService userService;
-    private static MessageRepository messageRepository;
+    private final UserService userService;
+    private final AssociationService associationService;
+    private final MessageRepository messageRepository;
     private final UserRepository userRepository;
     private final MessageSource messageSource;
     private final AttachmentService attachmentService;
 
-    public MessageService(UserService userService, MessageRepository messageRepository, UserRepository userRepository, MessageSource messageSource, AttachmentService attachmentService) {
-        MessageService.userService = userService;
```

Comment:
I took the liberty to put those into private finals instead of static as they are not used anywhere else in the codebase as well

Reply:
Ok. Idk why there was static.